python/nav/web/auth/remote_user.py

```diff
@@ -179,6 +182,10 @@ def get_username(request):
     if not request:
         return None
 
+    if settings.DEBUG or check_log_level(_logger, logging.DEBUG):
```

Comment:
Is this extra function really necessary? I find the following to be just as clear:
```suggestion
    if settings.DEBUG or _logger.getEffectiveLevel() <= logging.DEBUG:
```

Reply:
The function makes for slightly easier branch testing if one is fond of mocks and patches. Also it might prevent someone from prettifying `getEffectiveLevel()` to `level` or what it was, which does not do what one thinks it does.

Reply:
`settings.DEBUG` is probably not even necessary since that doesn't flip debug-logging magically on.

Reply:
Well, this jogged my memory, and I recall NAV already has multiple examples of the (to me) even more legible pattern:
```suggestion
    if settings.DEBUG or _logger.isEnabledFor(logging.DEBUG):
```